.buildkite/serve.rayci.yml

```diff
@@ -47,3 +71,14 @@ steps:
       - bazel run //ci/ray_ci:test_in_docker -- //... serve --run-flaky-tests --parallelism-per-worker 3
     depends_on: servebuild
     job_env: forge
+
+  - name: minbuild-serve
+    label: "wanda: minbuild-{{matrix}}-py38"
+    wanda: ci/docker/min.build.wanda.yaml
+    depends_on: oss-ci-base_build
+    matrix:
+      - serve
+      - default
+    env:
+      PYTHON_VERSION: "3.8"
```

Comment:
maybe set the default PYTHON_VERSION in the dockerfile or something? does not feel the right place to specify python version here.

or use buildkite matrix adjustment?

https://buildkite.com/docs/pipelines/build-matrix#adding-combinations-to-the-build-matrix

or just create another wanda yaml file?

Reply:
100%, there should be one DEFAULT_PYTHON constant in the code base, but we need a way to pass it through cross language first. Since the flow here is buildkite -> wanda -> docker -> python, the missing piece here is wanda to support default env variable ;). I'll keep this for now and handle this tech debt later with new feature from wanda template.

Creating another wanda is more problematic I think since they share the same content.